app/components/input-range.tsx

```
@@ -35,7 +35,8 @@ export function InputRange({
         max={max}
         step={step}
         onChange={onChange}
-      ></input>
+      >
+      </input>
```

Biome (1.9.4)

[error] input is a void element tag and must not have children.

Unsafe fix: Remove the children.


(lint/correctness/noVoidElementsWithChildren)